akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -929,6 +929,9 @@ class DDataShardCoordinator(typeName: String, settings: ClusterShardingSettings,
     if (rememberEntities) Set(CoordinatorStateKey, AllShardsKey) else Set(CoordinatorStateKey)
 
   var shards = Set.empty[String]
+
+  var getShardHomeRequests: Set[(ActorRef, GetShardHome)] = Set.empty
```

Comment:
I still haven't figured out how to specifically test for non-duplicated `GetShardHome` message.

I did run the `validatePullRequest` before creating this PR which invokes `akka-cluster-sharding` and `akka-cluster-sharding-typed`, and all tests seems to pass.

If anyone has any hint or ideas on how to test for non-duplicated `GetShardHome` message, would you kindly share them with me please? Or maybe it's not necessary to specifically test for this due to existing coverage in the `ClusterShardingSpec.scala`?

Any advice will be greatly appreciated - thanks!

Reply:
Perhaps:
* create Coordinator
* send `GetShardHome`, validate response
* make it go into a state where it needs to update
* send 2 `GetShardHome`s
* send the update
* expect exactly 1 response (then `expectNoMessage`)

Would that be possible? Or is there anything that makes that scenario hard to reconstruct in a test case?

Reply:
@raboof - sorry for the late response, I've been away for a couple of days.

I will try with your suggestion and report back here.